stubs/inertia-vue-ts/resources/js/Components/TextInput.vue

```diff
@@ -1,11 +1,7 @@
 <script setup lang="ts">
 import { onMounted, ref } from 'vue';
 
-defineProps<{
-    modelValue: string;
-}>();
-
-defineEmits(['update:modelValue']);
+const modelValue = defineModel();
```

Comment:
```suggestion
const modelValue = defineModel<string>();
```

I do not use Vue much, and this is untested and just from reading the docs: https://vuejs.org/api/sfc-script-setup.html#usage-with-typescript

Reply:
Maybe add here the required part to? Seems inconsistent with the non typescript part ?
```suggestion
const modelValue = defineModel<string>({ required: true });
```